prime-angular-frontend/src/app/modules/site-registration/pages/overview/overview.component.html

```diff
@@ -1,5 +1,6 @@
 <app-page [busy]="busy">
   <app-page-header>PharmaNet Site Registration</app-page-header>
+  <!-- <app-site-progress-indicator [inProgress]="!isCompleted"></app-site-progress-indicator> -->
```

Comment:
forgot commented code?

Reply:
Pilar wants the progress bar added to the Review screen but said that she will create a new Jira ticket for this.  I left it commented-out for now

Reply:
I think if there will be a separate ticket we should remove it from here so we aren't merging in commented code if avoidable.

Reply:
OK done